### PR TITLE
Hero image compressed and Color Palette changed

### PR DESCRIPTION
**Pull request by drakehere**
- compressing the hero image
- Updating the color palette for Docs from Green -> Blue

### CODE REVIEW

File: docs/docusaurus.config.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ const config = {
         playgroundPosition: "bottom",
       },
       navbar: {
+        hideOnScroll: true,
```
Comment (0xSage) [inline docs/docusaurus.config.js:113]: Let's not hide on scroll, bad ux.

File: docs/src/css/custom.css [hunk 2]
```diff
@@ -32,6 +32,5 @@
   --ifm-color-primary-light: #29d5b0;
```
Comment (0xSage) [inline docs/src/css/custom.css:32]: dark mode highlighting is still Docusaurus green. Is that what we want?

File: docs/static/img/desktop-explore-models-dark.png
Comment (0xSage) [inline docs/static/img/desktop-explore-models-dark.png:1]: Can we have a light mode

File: docs/static/img/desktop-explore-models.png
Comment (0xSage) [inline docs/static/img/desktop-explore-models.png:1]: Can we have a dark mode as well

File: docs/src/components/Homepage/hero.js [hunk 3]
```diff
@@ -39,21 +39,17 @@ export default function HomepageHero() {
         )}
 
         {/* Main hero block */}
-        <div className="py-24 sm:py-32 lg:pb-40">
+        <div className="py-24 sm:py-32 lg:pb-40 animate-in fade-in zoom-in-50 duration-1000 ">
```
Comment (0xSage) [inline docs/src/components/Homepage/hero.js:42]: Can we fix the glitch as a result of this animation. 
To repro:
1. hard refresh the site
2. observe glitch where copy shows up, then disappears, and animates up again.

File: docs/src/components/Homepage/hero.js [hunk 4]
```diff
@@ -39,21 +39,17 @@ export default function HomepageHero() {
...
               </h1>
               <p className="mt-6 text-lg leading-8 text-gray-600 dark:text-gray-300">
-                Jan lets you run AI on your own hardware. 1-click to install the
-                latest open-source models. Monitor and manage software-hardware
-                performance.
-                <br></br>
-                Jan is
-                <strong> free and open core</strong> with a Sustainable Use
-                License.
+                Anim aute id magna aliqua ad ad non deserunt sunt. Qui irure qui
```
Comment (0xSage) [inline docs/src/components/Homepage/hero.js:50]: Why are we deleting valid copy and adding invalid copy?

**Review by 0xSage — CHANGES_REQUESTED** [review]

File: docs/static/img/desktop-explore-models.png
Comment (drakehere) [reply docs/static/img/desktop-explore-models.png:1]: Dark mode screenshot has been added

Screenshot 2023-10-02 at 16 24 01

**Comment by drakehere** [comment]
New updates:
- Dark and Light mode for each screenshot
- Hero section's copies are back
- Dark mode now have link and button highlighted in Blue

https://www.loom.com/share/98f9c40d904a4dcfb37e4c6a7cadfc7d?sid=8d9cff48-0325-4614-a204-971e972df3a7

**Review by 0xSage — APPROVED** [review]

**Comment by 0xSage** [comment]
@drakehere approved, pls feel free to merge

**Pull request merged**